web/src/services/api/projectApi.ts

```diff
@@ -387,6 +389,102 @@ export default () => {
     [updateProjectAliasMutation, t, setNotification]
   );
 
+  const [exportProjectMutation] = useMutation(EXPORT_PROJECT);
+
+  const useExportProject = useCallback(
+    async (projectId: string) => {
+      if (!projectId) return { status: "error" };
+
+      try {
+        const { data, errors } = await exportProjectMutation({
+          variables: { projectId }
+        });
+
+        if (errors || !data?.exportProject?.projectDataPath) {
+          console.log("GraphQL: Failed to export project", errors);
+          setNotification({
+            type: "error",
+            text: t("Failed to export project.")
+          });
+
+          return { status: "error" };
+        }
+
+        const projectDataPath = data.exportProject.projectDataPath;
+
+        const backendUrl = window.origin.replace("3000", "8080");
```

Comment:
I'm not quite sure, but I think it's necessary to refarence the environment variables.

Local environment:
[UI] http://localhost:3000/...
[API] http://localhost:8080/...

Dev/Prod environment:
[UI] https://visualizer.test.reearth.dev/...
[API] https://api.test.reearth.dev/export/...

Reply:
Yes i think we can use `config().api` to get the entrance of backend, and remove the `/api` in the end.